Dialogue/lordgraybeard.tres

```diff
@@ -0,0 +1,1418 @@
+Lord Graybeard: I need you to go to the distant crypt known as VarLog and take care of it.
+=> quest_1_responses
+
+~ quest_1_responses
+- Why are the logs in a crypt?
+	Lord Graybeard: It's where they go to die, of course
+	=> quest_1_responses
+- If I remove them, won't they just come back?
+	Lord Graybeard: Most likely. We must find the source, but for now, deleting them should buy us some time.
+	Lord Graybeard: If this gets any worse, the root filesystem will fill up and then we'll be in big trouble!
+	=> quest_1_responses
+- I'm ready to go!
+	=> quest_1_part_2
+- I don't bother will quest text, just send me on my way!
+	=> quest_1_part_2
+	
+
+~ quest_1_part_2
+Lord Graybeard: Once you've destroyed all the logs in the VarLog, come back to me.
+do set_quest_state(\"any_quest\", \"state\", \"in_progress\")
+do set_quest_state(\"quest_1\", \"state\", \"in_progress\")
+if get_dialogue_state(\"graybeard\", \"kind\") == true
+	Hero: Sounds good! I'm on it
+	Lord Graybeard: Good luck!
+	=> END
+else:
+	Hero: So I can get paid?
+	Lord Graybeard: Fine, yes yes. You'll get your hours. Now go!
+    => END
+
+~ active_quest_meet
+Lord Graybeard: Oh hello again!
+if get_quest_state(\"quest_1\", \"state\") == \"in_progress\"
+	Lord Graybeard: Shouldn't you be out destroying log files in VarLog?
+	Hero: But I have questions
+	=> questions
+else
+	Lord Graybeard: Hello again. Can I help you?
+	- I need more quests.
+	    Lord Graybeard: I'm afraid this demo server is limited.
+	    Hero: But I only did one!
+	    Lord Graybeard: Severely limited? Come back later.
+	    => END
+	- I have more questions
+	    Lord Graybeard: Very well
+	    => questions
+	
+~ questions
+- What's the deal with all the different styles around here?
+	Lord Graybeard: It's an old system. Many a sysadmin has been through here over the years. Each one leaving their mark.
+	Lord Graybeard: My programming has been here for ten years. Untouched. 
+	Hero: Why are you still here then?
+	Lord Graybeard: No one knows what I do so they're afraid to turn me off.
+	Hero: That checks out.
+	=> questions
+- What do I do with my billable hours?
+	Lord Graybeard: You can buy stuff with it!
+	Hero: Where?
+	Lord Graybeard: Traceback: NameError: \"Shop\" is not defined
+	Hero: What?
+	Lord Graybeard: Looks like no one put in the code for it yet.
+	=> questions
+- What language are you written in?
+	Lord Graybeard: Python
+	Hero: Oh, well that's not so bad.
+	Lord Graybeard: Python 2
+	Hero: Nevermind
+	=> questions
+- Nevermind
+	Lord Graybeard: Good! I need a nap anyway
+	=> END
+
+~ turn_in
+Lord Graybeard: All done then?
+Hero: You bet!
+Lord Graybeard: Well good! I guess the system is in better shape than it was.
+Lord Graybeard: I can feel it in my bits.
+Hero: Gross.
+Lord Graybeard: Anyway, here are your hours. Now go home!
+# need to have a function call to award hours and end the game"
```

Comment:
`increase_hours(100)` should do. It's part of the game_state

Reply:
Great! I'll add that. man, that's convenient